frontend/views/containers/global-dashboard/NewsAndUpdates.vue

```diff
@@ -152,18 +32,80 @@ export default ({
   },
   data () {
     return {
-      dummyPosts
+      posts: [],
+      isLoading: true,
+      error: false,
+      errorMessage: ''
     }
   },
+  computed: {
+    ...mapGetters(['ourIdentityContractId', 'currentIdentityState'])
+  },
+  async mounted () {
+    await this.fetchNews()
+    await this.markNewsAsSeen()
+  },
   methods: {
-    humanDate
+    humanDate,
+    async fetchNews () {
+      try {
+        this.isLoading = true
+        this.error = false
+
+        const response = await fetch('https://groupincome.org/news.json')
+        if (!response.ok) {
+          throw new Error(`HTTP error! status: ${response.status}`)
+        }
+
+        const data = await response.json()
+
+        // Convert createdAt strings to Date objects for proper formatting
+        this.posts = data.map(post => ({
+          ...post,
+          createdAt: new Date(post.createdAt)
+        }))
+      } catch (error) {
+        console.error('Failed to fetch news:', error)
+        this.error = true
+        this.errorMessage = error.message || 'Unknown error'
```

Comment:
Remember to wrap all human-facing strings in the `L` function, so `Unknown error` needs to be wrapped here.